lib/usd/ui/layerEditor/layerTreeItem.cpp

```diff
@@ -32,8 +32,8 @@ LayerActionDefinitions LayerTreeItem::_actionButtons;
 
 static void createPixmapPair(const QString& name, QPixmap& normal, QPixmap& hover)
 {
-    normal = utils->createPNGResPixmap(utils->getDPIPixmapName(name));
-    hover = utils->createPNGResPixmap(utils->getDPIPixmapName(name + QString("_hover")));
+    normal = utils->createPNGResPixmap(name);
+    hover = utils->createPNGResPixmap(name + QString("_hover"));
```

Comment:
So when user is running at 200% how will it pick the _200.png file?
It seems like with these changes it will always use the 100% file and scale that?

Reply:
The function createPNGResPixmap calls createPixmap which modifies the image name to add the scaling value to it (_150, _200) when necessary.

Reply:
createPNGResPixmap calls QtUtils::createPNGResPixmap  which calls MayaQtUtils::createPixmap which calls MQtUtil::createPixmap which calls QmayaQtHelper::createPixmap which generates the scaled image name by adding the _150 or _200 suffix as necessary.

What is somewhat misleading is that there is a QtUtils::createPixmap function, but that funciton is virtual, so that is not the one that gets called, instead it calls the MayaQtUtils version which does the magic.

Reply:
Maybe I could put a comment in createPNGResPixmap with these explanations?

Reply:
Ah yes I missed that overridden function. Thanks for the explanation. Yeah maybe a comment in "createPNGResPixmap" would be nice, saying that when running in Maya the proper png (based on scaling) will be used.

Reply:
No worry, when you asked, I knew the answer because I had traced the execution, but when I went to read the code I was as confused as you and had to re-run the code to figure out what was going on...